fakebook/src/app/model/user.ts

```diff
@@ -7,4 +7,6 @@ export interface User {
   profilePictureUrl: string | null;
   status: string | undefined;
   birthDate: Date;
+  followers: User[];
+  followees: User[];
```

Comment:
be sure backend and frontend aren't out of sync on the data format.

Reply:
We checked with the backend so they know now. The followers and followees are tracked by the post service according to Jay